src/environmentd/src/http/sql.rs

```diff
@@ -14,16 +14,37 @@ use serde::Deserialize;
 
 use crate::http::AuthedClient;
 
-#[derive(Deserialize)]
+#[derive(Deserialize, Debug)]
+pub struct SqlRequestParam {
+    value: String,
+}
+
+#[derive(Deserialize, Debug)]
 pub struct SqlRequest {
-    sql: String,
+    query: String,
```

Comment:
Would be great to preserve the old name, at least for a little, to give the cloud UI time to catch up:

```suggestion
    #[serde(alias = "sql")]
    query: String,
```